Shared/PathUtils.cs

```diff
@@ -131,5 +131,21 @@ public static string GetTempFileName(string extension)
             //use instead of Path.GetTempFileName()
             return Path.Combine(Path.GetTempPath(), Path.GetRandomFileName() + extension);
         }
+
+        /// <summary>
+        /// Checks if the relative path has any ".OpenTapIgnore" file in parent directory chain. Throws argument error eventually if input is an absolute path and no ".OpenTapIgnore" file is present in parent directory chain.
+        /// </summary>
+        /// <param name="location">Relative path of file</param>
+        /// <returns>Whether an .OpenTapIgnore file exists in folders.</returns>
+        internal static bool DecendsFromOpenTapIgnore(string location)
```

Comment:
Should we prevent this method from recursing out of the install directory? `touch C:\.OpenTapIgnore` is going to brick everything ever and is difficult to debug

Reply:
This method explicitly states that it takes a relative path, so this should not be a problem. Examples of inputs: "Packages/REST-API/xxx.dll"

Reply:
One thing I could do to make this clearer is the input argument to actually be named "relativePath" instead of "location". Would this help?